modular_zubbers/code/modules/projectiles/projectile/pistol.dm

```diff
@@ -0,0 +1,6 @@
+/obj/projectile/bullet/c9mm/security
+	name = "9mm Security bullet"
+	damage = 20
+	embed_type = /datum/embedding/bullet/c9mm
+	weak_against_armour = FALSE
```

Comment:
/obj/projectile/bullet/c9mm does not have weak to armor, why are you setting it to false and giving it the same embed type it already has? Probably could use a better name than just security perhaps as well.